docs/admin/support.rst

```diff
@@ -60,37 +60,37 @@ Discover Weblate
 
    This feature is currently in early beta.
 
-Discover Weblate is an opt-in service that makes it easier for users to find
-Weblate servers and communities. Users can browse registered services on
-<https://weblate.org/discover/>, and find there projects to contribute.
+Discover Weblate is an opt-in service making it easier for users to find
+other Weblate sites and communities. Users can browse registered sites on
+https://weblate.org/discover and find
+projects to contribute to there.
 
 Getting listed
 ++++++++++++++
 
-.. hint::
+.. note::
 
-   Participating in Discover Weblate makes Weblate submit some information
-   about your server, please see :ref:`support-data`.
+   Make sure you are OK with publishing the info about your Weblate
+   described in :ref:`support-data` for use on Discover Weblate.
 
 
-To list your server with an active support subscription (see
-:ref:`activate-support`) in Discover Weblate all you need to do is turn this on
-in the management panel:
+List your server directly from the management panel if you have an active
+support subscription (:ref:`activate-support`):
 
 .. image:: /images/support-discovery.png
 
-Listing your server without a support subsription in Discover Weblate:
+Listing your server manually:
```

Comment:
This is not a more manually that with a subscription, it just needs additional steps.